### PR TITLE
Dnsproviders as extension

### LINKED ISSUE #1956: Move DNS Providers as extension
**Description**

We replace our current implementation with the extensibility option.

**Acceptance Criteria**

- [ ] Validate with Product Owner if the current implementation is correct
- [ ] Replace the current implementation with extensibility
- [ ] Deploy to stage and Test all the edge cases on stage
- [ ] Deploy to production
- [ ] Remove old codebase
- [ ] Adjust Tests to use extensions

### PR DESCRIPTION
**Pull request by chriskari**
**Description**

Changes proposed in this pull request:

- added dnsproviders as extension
- adjusted tests
- removed old codebase

**Related issue(s)**
closes #1956

### CODE REVIEW

File: tests/tests/namespace/test-dns-providers.spec.js [hunk 1]
```diff
@@ -91,11 +103,11 @@ context('Test DNS Providers', () => {
...
-    // edit excluded domains
-    cy.get('[role=dialog]')
-      .contains('Include Domains')
-      .filter(':visible', { log: false })
-      .click();
+    // edit included domains
+    //cy.get('[role=dialog]')
+    //  .contains('Include Domains')
+    //  .filter(':visible', { log: false })
+    //  .click();
```
Comment (mrCherry97) [inline tests/tests/namespace/test-dns-providers.spec.js:110]: This should be removed

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
After our changes to testing part of busola and kyma-dashboard, test should be moved to kyma-dashboard and extension should be removed from busola codebase.

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
Remove `test-dns-providers.spec.js` from `cypress.json`

**Review by mrCherry97 — APPROVED** [review]

**Review by nataliasitko — APPROVED** [review]
